### ISSUE TITLE
feature: add a `@file` operator

### ISSUE THREAD

**Issue by tusharmath** [tailcallhq/tailcall#932]
Add a `@file` operator like `@http` to read data from a file.

```graphql
schema {
  query: Query
}

type Query {
  user: User @file(src: "./users.json")
}
```

**Technical Requirements:**
- The file should be read only once at the time of starting the server.
- Check should not validate if the file exists.
- File path should be relative to the config.
- File format should be detected via file extension.
- Support - YAML, JSON for now.

**Comment by ayewo** [comment]
✋

**Comment by arvandmoe** [comment]
@tusharmath I'd like to work on this.

**Comment by ologbonowiwi** [comment]
Is this available? I'd like to /attempt #932
